src/main/java/com/google/javascript/gents/RemoveGoogScopePass.java

```diff
@@ -93,40 +100,50 @@ private void rewriteGoogScope(Node n) {
   }
 
   private Node maybeRewriteAlias(Node node) {
+    // Return the next node if the current node is detached.
```

Comment:
Can you flesh this out into documentation for the method and put it in a jsdoc ?